7_react/react1/week1/src/todo_list/TodoList.jsx

```diff
@@ -0,0 +1,51 @@
+import { TodoItem } from "../todoItem/TodoItem";
+const AppTodo = () => {
+    
+    const todos = [
+      {
+        name: "Get out of bed",
+        deadline: "Wed Sep 13 2017",
+        status: true,
+      },
+      {
+        name: "Brush teeth",
+        deadline: "Thu Sep 14 2017",
+        status: false,
+      },
+      {
+        name: "Eat breakfast",
+        deadline: "Fri Sep 15 2017",
+        status: false,
+      },
+    ];
+
+   
+
+    const TodoList = (props) => {
```

Comment:
Hey @AlesyaMazurenko Please move `TodoList` component to its separate file, then you can export it with `export` keyword